recipe/meta.yaml

```diff
@@ -8,35 +8,32 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 4e913d807aa6598c320e8a50c50e2ee0602bc45240b485e3f8bc06f13060084c
+  sha256: 21f535de2e7a82c94c130c054adb5c7421d480d5619d61073996e2f89bcb879b
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: true  # [py<39]
 
 requirements:
   host:
     - python
     - pip
     - pytest-runner
+    - setuptools
+    - setuptools_scm
+    - wheel
   run:
     - python
-    - pywin32  # [win]
+    - pywin32 >=226  # [win]
 
 test:
   imports:
     - portalocker
-  source_files:
-    - portalocker_tests
   requires:
-    - pytest >=3.4.0
-    - pytest-cache >=1.0
-    - pytest-cov >=2.5.1
-    - pytest-flakes >=2.0.0
-    - pytest-pep8 >=1.0.6
-  # TODO: how to skip just test_combined ?
-  # commands:
-  #  - pytest -k "not test_combined" portalocker_tests
```

Comment:
`pytest --ignore=test_combined.py -v portalocker_tests` should work. That or something like `pytest --ignore=portalocker_tests/test_combined.py -v portalocker_tests`

Reply:
added tests back in!